pkg/syncer/k8scloudoperator/placement.go

```diff
@@ -53,6 +53,9 @@ const (
 	resourceName = "storagepools"
 	// storagePool type name for vsan-direct
 	vsanDirect = "vsanD"
+	INVALID_PARAMS = "FAILED_PLACEMENT-InvalidParams"
```

Comment:
Just curious about the pattern in these constants.
why we have some chars capitalized and regular camelcase. Any special reason?